values.yaml

```diff
@@ -327,8 +327,19 @@ airflow:
       elasticsearch_write_stdout: True
       elasticsearch_json_format: True
       elasticsearch_log_id_template: "{dag_id}_{task_id}_{execution_date}_{try_number}"
+    # The following kubernetes config is required to support Airflow 1.10.10
```

Comment:
For future readers -- Airflow 2.x will ignore these configs so there are here just for backwards compatibility